exts/help.py

```diff
@@ -0,0 +1,239 @@
+                ctx.responses["help_thread_name"],
+                content=message.content,
+                applied_tags=[METADATA["autogenerated_tag"]],
+                auto_archive_duration=1440,  # type: ignore
+                files=files,  # type: ignore
+                reason="Auto help thread creation",
+            )
+
+            await post_thread.add_member(ctx.author)
+            await post_thread.add_member(message.author)
+
+            embed = None
+
+            if content := ctx.responses.get("extra_content"):
+                embed = naff.Embed(
+                    title="Additional Information",
+                    description=content,
+                    color=ASTRO_COLOR,
+                )
+                embed.set_footer(text="Please create a thread in #help to ask questions!")
+
+            select = self.generate_tag_select()
+
+            original_message_button = naff.Button(
+                style=naff.ButtonStyles.LINK,
+                label="Original message",
+                url=message.jump_url,
+            )
+            close_button = naff.Button(
+                style=naff.ButtonStyles.DANGER,
+                label="Close this thread",
+                custom_id="close_thread",
+            )
+
+            starter_message = await post_thread.send(
+                (
+                    "This help thread was automatically generated. Read the message above for more"
+                    " information."
+                ),
+                embeds=embed,
+                components=[[original_message_button], [select], [close_button]],
+            )
+            await starter_message.pin()
+
+            await message.reply(
+                f"Hey, {message.author.mention}! At this time, we only help with support-related"
+                f" questions in our help channel. Please redirect to {post_thread.mention} in order"
+                " to receive help."
+            )
+            await ctx.send(":white_check_mark: Thread created.", ephemeral=True)
+
+    @naff.listen("new_thread_create")
+    async def first_message_for_help(self, event: naff.events.NewThreadCreate):
+        thread = event.thread
+        if not thread.parent_id or int(thread.parent_id) != METADATA["channels"]["help"]:
+            return
+
+        if thread.owner_id == self.bot.user.id:
+            # an autogenerated thread, don't interfere
+            return
+
+        select = self.generate_tag_select()
+        close_button = naff.Button(
+            style=naff.ButtonStyles.DANGER,
+            label="Close this thread",
+            custom_id="close_thread",
+        )
+
+        message = await thread.send(
+            "Hey! Once your issue is solved, press the button below to close this thread!",
+            components=[[select], [close_button]],
+        )
+        await message.pin()
+
+    @naff.component_callback("TAG_SELECTION")  # type: ignore
+    async def modify_tags(self, ctx: naff.ComponentContext):
```

Comment:
Are we ditching this in favour of perms v2?

Reply:
Like, manual checks? Most likely, but I found it a bit dangerous to have advanced/helper mmands be used by anyone, even if it's only a second. I can remove the checks afterwards... though I guess another option is making it have a `default_member_permissions` value, and then people with admin perms can modify it from there to give permissions to helpers.

Reply:
I think having default member perms would be best

Reply:
And done. Of course, there's only one command that benefits from it, but oh well.